src/gfx/materials.js

```diff
@@ -94,13 +90,11 @@ const materialList = [
       depthWrite: true,
       transparent: true,
       toonShading: false,
-      side: THREE.DoubleSide,
     },
   }, {
     id: 'GL',
     name: 'Glass',
     shortName: 'Glass',
-    depthWrite: true,
```

Comment:
Why have you removed the depthWrite property from the material?

Reply:
In Glass material "depthWrite: true" was written twice/ So I removed one of them (which was written in the wrong place - depthWrite usually defined in "values" section)